src/components/our-vision-story-blocks.js

```diff
@@ -24,14 +25,17 @@ const OurVisionStoryBlocks = () => {
       }
     }
   `)
+  const intl = useIntl()
   const storyData = storysQuery.contentJson.expanded_stories
   return (
     <div className="our-vision-text">
       {storyData.map((node, index) => (
         <StoryBlock
           imageSource={node.image.childImageSharp.fluid}
           altText={node.altText && node.altText}
```

Comment:
this one needs to be updated too. we can leave the image ones alone for now, since they dont change if the language changes